lib/avalon/batch/manifest.rb

```diff
@@ -82,19 +124,28 @@ def load!
 
       def start!
         File.open("#{@file}.processing",'w') { |f| f.puts Time.now.xmlschema }
+        manifest_logger.info("Processing started #{@file}")
```

Comment:
If your displaying the manifest logger to the end user, should every message to this logger be ran through i18n?